### ISSUE TITLE
can samblaster tackle reads with UMI?

### ISSUE THREAD

**Issue by windtalker6** [GregoryFaust/samblaster#47]
can samblaster tackle reads with UMI?

**Comment by GregoryFaust** [comment]
Yes and No. Yes in the sense that *samblaster* will treat SAM files containing UMIs like any other SAM file. No in the sense that *samblaster* does not restrict identification of duplicate pairs to reads from the same molecule. Handling UMIs has been a previous request that is on the top of my priority list to address. See issue #20.
